### PR TITLE
New Presence | MangaTube

### PR DESCRIPTION
**Pull request by GitEmir**
i made a MangaTube presence
preview images:

Viewing Homepage | 
bild5

Viewing Manga |
bild2

Reading Manga |
bild3

Manga List |
bild4

Viewing Profile
bild1

do i have to prove everything or is that enough?

### CODE REVIEW

File: websites/M/MangaTube/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Emir",
+      "id": "466885774621540372"
+    },
+    "url": "manga-tube.me",
+    "description": {
+      "de": "Deine deutsche Manga-Community. Unterhalte dich mit vielen gleichgesinnten Manga-Fans und wühle dich durch unsere riesige Datenbank mit über 600 Manga",
+      "en": "none"
```
Comment (Bas950) [inline websites/M/MangaTube/dist/metadata.json:10]: You need an English description

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors and remove unnecessary comments

**Comment by GitEmir** [comment]
sry for that, i am kinda stupid today

File: websites/M/MangaTube/dist/metadata.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+      "id": "466885774621540372"
+    },
+    "url": "manga-tube.me",
+    "description": {
+      "de": "Deine deutsche Manga-Community. Unterhalte dich mit vielen gleichgesinnten Manga-Fans und wühle dich durch unsere riesige Datenbank mit über 600 Manga",
+      "en": "Your German Manga Community. Talk with Manga fans and browse through our huge database with over 600 Manga"
+    },
+    "service": "MangaTube",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/SJlxyjc.png",
```
Comment (Bas950) [inline websites/M/MangaTube/dist/metadata.json:14]: please get a better quality logo

**Review by Bas950 — CHANGES_REQUESTED** [review]
And you will probs get more codacy errors.

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**